app/js/main.js

```diff
@@ -1097,4 +1183,15 @@ function loadMenu() {
     if (process.platform === "darwin") {
         menuBar.createMacBuiltin("Boats Animator");
     }
-}
\ No newline at end of file
+}
+
+// Stops globsl shortcuts operating in other applications
+win.on("focus", function() {
```

Comment:
For all three of these events, `win.on("<event>", shortcut.methodName);` might work.